docs/user-guide/configuration-files/strongbox-yaml.md

```diff
@@ -202,6 +210,7 @@ The following is an example of a `strongbox.yaml` configuration file:
                 downloadRemoteIndexes: true
                 autoBlocking: true
                 checksumValidation: true
+                metadataStrategy: checksum
```

Comment:
@cbrianhill : I've just realized that this functionality should perhaps be Maven-specific. I mean these settings will make no sense in the context of Pypi, NPM, or Nuget. 

@strongbox/core-developers : What do you guys think?

Reply:
I believe this is just a typo - the code from strongbox/strongbox#1491 has this field in the `MavenRepositoryConfigurationDto.java` so it should be available only under the maven repositories, right?  :)

Reply:
@steve-todorov you're right, this is a typo.  Will fix.  Also realized I need to update this with the finalized name of the property.